rails-buildpack/2.7.5/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ruby:2.7.5
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+      file \
+      git \
+      unzip \
+      curl \
+      autoconf \
+      automake \
+      default-libmysqlclient-dev \
+      default-mysql-client \
+      g++ \
+      gcc \
+      gnupg \
+      patch \
+      make \
+      libbz2-dev \
+      libc6-dev \
+      liblzma-dev \
+      libmagickcore-dev \
+      libmagickwand-dev \
+      libreadline-dev \
+      libtool \
+      libxslt-dev \
+      libpq-dev \
+      libsqlite3-dev \
+      libxml2-dev \
+      qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools \
```

Comment:
`ruby:2.7.5` is a full image that's usually [based on](https://medium.com/swlh/alpine-slim-stretch-buster-jessie-bullseye-bookworm-what-are-the-differences-in-docker-62171ed4531d) the latest stable Debian OS. It seems from at least Debian 10.4 release, `qt4default` [has been deprecated](https://packages.debian.org/buster/qt5-default) in favor of more explicit dependencies build.